open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/FileBasedServerConfigStoreConnector.java

```diff
@@ -144,7 +153,27 @@ public void removeServerConfig()
 
         configStoreFile.delete();
     }
-
+    @Override
+    public Set<OMAGServerConfig> retrieveAllServerConfigs() {
```

Comment:
Code formatting

Can not just test for files that end in config - need to use the file formatting string.

Reply:
Good spot -I was not aware of that.

Reply:
changed the code to use the formatting.